src/xdp/ebpf_private_extension.h

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+
+#pragma once
+
+//
+// The following definitions are from ebpf-for-windows but are not defined in
+// its "include" directory.
```

Comment:
If these are needed, is there an issue filed in the ebpf-for-windows repo?

Reply:
Not yet - I'll file issues for this and other comments once this PR stabilizes.

Reply:
Did you create the issue for this? Could you update the comment to point to it so we can follow up when it's done?